cmd/soroban-cli/src/commands/plugin/ls.rs

```diff
@@ -0,0 +1,22 @@
+use super::default;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {}
+
+#[derive(Debug, clap::Parser, Clone)]
+#[group(skip)]
+pub struct Cmd;
+
+impl Cmd {
+    pub fn run(&self) -> Result<(), Error> {
+        let plugins = default::list().unwrap_or_default();
+
+        if plugins.is_empty() {
+            println!("No Plugins installed. E.g. stellar-hello");
```

Comment:
The example doesn't feel like it has enough context. Suggest we add:
```suggestion
            println!("No Plugins installed. Plugins are commands available on the path that start with 'stellar-'. E.g. stellar-hello");
```

Reply:
Good call!

Changed it to give this output instead:

```console
$ stellar --list
No plugins installed.

Plugins are commands available on the path
that start with 'stellar-'. E.g. stellar-hello

https://developers.stellar.org/docs/tools/cli/plugins
```